logger/mmap_processor.cc

```diff
@@ -0,0 +1,321 @@
+/**
```

Comment:
"mmap" is just a way to write events and other logs may use the same technique, so it'd be better to rename the couple of files to something like `access_log_processor.[ch][ch]`